virtualization.go

```diff
@@ -16,10 +16,6 @@ import (
 	"github.com/Code-Hex/vz/v3/internal/objc"
 )
 
-func init() {
-	C.sharedApplication()
```

Comment:
Actually, it should be called in the main thread before other Objective-C code, but since I did not specify the main thread, I thought it was safe to remove it.